Add the 1.18 milestone tracking sheet link to README

palnabarun opened the pull request:
/assign

palnabarun commented:
/sig pm
/sig release
/cc @jeremyrickard

jeremyrickard commented:
/lgtm

palnabarun commented:
/assign @justaugustus

justaugustus commented on README.md:
```diff
@@ -62,6 +62,7 @@ Please do not comment on the enhancement issue to:
 All the enhancements from this repo are visualized in the Enhancements Tracking Spreadsheets.
 
 Links:
+- [1.18 Milestone](https://docs.google.com/spreadsheets/d/1RtCvByYdcqWc6I_A1cKgeXT2tBS7SyHGvSt_DWXz270)
```
Ideally, all of these should instead be the bit.ly shortlinks we set up for each release.
e.g., this cycle --> http://bit.ly/k8s-1-18-enhancements

justaugustus commented on README.md:
```diff
@@ -74,10 +75,10 @@ Procedure:
-| `tracked/yes` | Denotes an issue has been reviewed by a Feature Maintainer (SIG PM / SIG Release) and is actively tracked for the current milestone | Manually set | Feature Maintainers (SIG PM / SIG Release) ONLY |
-| `tracked/no` | Denotes an issue has been reviewed by a Feature Maintainer (SIG PM / SIG Release) and will not actively tracked for the current milestone | Manually set | Feature Maintainers (SIG PM / SIG Release) ONLY |
-| `stage/{alpha,beta,stable}` | Denotes the stage of an issue in the features process. | Set the label using the comment `/stage alpha` (on a separate line) | Anyone |
+| Label Name                  | Purpose                                                                                                                                   | How to use this label                                                | Who should use this label                       |
```
Please drop the space additions. It suggests that we need to update the spacing every time we change the width of a column and that is automatically figured out when the document is rendered in markdown.

palnabarun replied:
Agreed. 

I added that as the document is also readable when opened in raw format.

justaugustus commented:
@palnabarun -- Any update here? Can you fix up the things I made comments on?

palnabarun commented:
@justaugustus -- Made changes according to the comments.

- Reverted the commit which beautified the tables.
- Added short link to the current enhancement tracker
- Modified existing tracking sheet links to their shortened URL's

justaugustus commented:
Thanks @palnabarun!
/lgtm
/approve
